zerver/lib/actions.py

```diff
@@ -822,6 +822,10 @@ def do_send_messages(messages_maybe_none):
                 if user_profile.id in mentioned_ids:
                     trigger = "mention"
 
+                # PM triggers for personal and huddle messsages
+                if message['message'].recipient.type != Recipient.STREAM:
+                    trigger = "personal_message"
```

Comment:
Style nit: would it make sense for this to be "private_message" (since it includes both personal and huddle messages)?